ex2_system/source/logger/logger.c

```diff
@@ -27,8 +27,13 @@
 #include <string.h>
 #include <redposix.h>
 
+#define XSTR_(X) STR_(X)
+#define STR_(X) #X
+
+
+
 #define DEFAULT_INPUT_QUEUE_LEN 10
-#define TASK_NAME_SIZE 13
+#define TASK_NAME_SIZE configMAX_TASK_NAME_LEN + 3

```

Comment:
Shouldn't be `configMAX_TASK_NAME_LEN - 3` ?

Reply:
The 3 is the brackets and null terminator, so I don't believe so

Reply:
Ok, I saw the previous value was 13 and configMAX_TASK_NAME_LEN is 16. I really don't know what their values should be :D

Reply:
Me neither, that's why I made it work with any value